### ISSUE TITLE
Confusion in regtools junction extract output

### ISSUE THREAD

**Issue by sagnikbanerjee15** [griffithlab/regtools#105]
Hello,

I have a bam file which has XS tags for spliced alignments. I used regtools to extract junction information. For some of the junctions, the strand information is ?. Could you please explain why that is so even when the strand information is present.

Thank you.

**Comment by yang-yangfeng** [comment]
Hi @sagnikbanerjee15 - could you post your command and version info?

**Comment by yang-yangfeng** [comment]
@sagnikbanerjee15, sorry if this was not clear in the documentation - if you want to use the XS tags, you should use the -s 0 option - otherwise regtools will by default assume you have RF data and interrogate the reads to infer strandness.

**Comment by yang-yangfeng** [comment]
@sagnikbanerjee15, I'm assuming this is no longer an issue for you - we've also removed the default setting - please let me know if you still need help!

**Comment by QuanLG** [comment]
Hi,I have a data from stranded library (fr-firststrand),then I used the hisat2 to map the reads and have the XS tags for each read. I used regtools to extract junction information. 
If I've not misunderstood ，it's the right that I set the -s 0 for use the XS tags？

**Comment by santataRU** [comment]
> Hi,I have a data from stranded library (fr-firststrand),then I used the hisat2 to map the reads and have the XS tags for each read. I used regtools to extract junction information. If I've not misunderstood ，it's the right that I set the -s 0 for use the XS tags？

I have the same question. I'm using regtools 0.5.0 for data from HISAT2, and I know the strandedness of my RNA-seq data is fr-firststrand/RF. According to the regtools options, I tried using `regtools junctions extract -s 1`. In the 6th column of the output .bed file, I see some junctions labeled with "+", some with "-", and some with "?".

I assumed that splice sites from the same gene should be in the same orientation, meaning that these pluses and minuses should mostly occur in groups of consecutive pluses or minuses. However, I observe an apparently random alternation of + and - in the 6th column of the bed file.

I also tried using `regtools junctions extract -s 0`, but I see similar patterns with some junctions labeled with "+", some with "-", and some with "?". I'm unsure what the correct approach is in this case.
